docs/xdr/features/collect/integrations/application/nginx.md

```diff
@@ -39,6 +42,13 @@ $InputRunFileMonitor
 
 The reader can consult the [Rsyslog Transport](../../../ingestion_methods/rsyslog/) documentation to forward these logs to Sekoia.io.
 
+### NGINX Configuration to forward logs in syslog UDP
+To configure NGINX to forward logs to an IP with syslog and the UDP protocol, please add these lines in `/etc/nginx/nginx.conf`:
```

Comment:
Un peu de contexte p-e ? On comprends pas trop quel est l'interet. Il faudrait p-e repréciser ce que tu as mis plus haut "send them directly to Sekoia.io or through a concentrator" avec un lien vers la doc de notre concentrator